package-lock.json

```diff
@@ -4086,6 +4092,16 @@
       "resolved": "https://registry.npmjs.org/binaryextensions/-/binaryextensions-2.3.0.tgz",
       "integrity": "sha512-nAihlQsYGyc5Bwq6+EsubvANYGExeJKHDO3RjnvwU042fawQTQfM3Kxn7IHUXQOz4bzfwsGYYHGSvXyW4zOGLg=="
     },
+    "bindings": {
```

Comment:
revert

Reply:
I tried to remove these dependencies but the pipeline then breaks and it can't go pass `npm install`. I can look into it further, but these dependencies were not added by me (maybe they were added by `file-saver` or other dependency that was externally updated).